agent/proto/agent.proto

```diff
@@ -295,6 +295,42 @@ message Timing {
     Status status = 6;
 }
 
+message GetResourcesMonitoringConfigurationRequest {
```

Comment:
This message is based on what has been described in the RFC - and has been mutually discussed and approved by Vincent and @DanielleMaywood